object.c

```diff
@@ -4565,6 +4565,143 @@ InitVM_Object(void)
+     *                                     based on the given arguments.
+     * - {#String}[#method-i-String]:: Returns a String based on the given argument.
+     *
+     * === Querying
+     *
+     * - {#__callee__}[#method-i-__callee__]:: Returns the called name
+     *                                         of the current method as a symbol.
+     * - {#__dir__}[#method-i-__dir__]:: Returns the path to the directory
+     *                                   from which the current method is called.
+     * - {#__method__}[#method-i-__method__]:: Returns the name at the definition
+     *                                          of the current method as a symbol.
+     * - #autoload?:: Returns the file to be loaded when the given module is referenced.
+     * - #binding:: Returns a Binding object that describes the variable and method
+     *              bindings at the point of call.
+     * - #block_given?:: Returns true if yield would execute a block
+     *                   in the current context.
+     * - #caller:: Returns the current execution stack as an array of strings.
+     * - #caller_locations:: Returns the current execution stack as an array of strings.
+     * - #class:: Returns the class of +self+.
+     * - #frozen?:: Returns whether +self+ is frozen.
+     * - #global_variables:: Returns an array of global variables as symbols.
+     * - #local_variables:: Returns an array of local variables as symbols.
+     * - #test:: Performs specified tests on the given single file or pair of files.
+     *
+     * === Executing
+     *
+     * - #`cmd`:: Returns the standard output of running +cmd+ in a subshell.
+     * - #eval:: Evaluates the Ruby code in the given string.
+     * - #exec:: Executes the given command.
+     * - #syscall:: Calls the operating system function identified by the given number;
+     *              returns the result or raises an exception if the function failed.
+     * - #system:: Executes the given command in a subshell.
+     *
+     * === Exiting
+     *
+     * - #abort:: Aborts the current process after printing the given arguments.
+     * - #at_exit:: Registers the given block as a proc to be executed
+     *              when the program exits.
+     * - #exit:: Exits the current process after calling any registered
+     *           +at_exit+ handlers.
+     * - #exit!:: Exits the current process without calling any registered
+     *            +at_exit+ handlers.
+     *
+     * === Exceptions
+     *
+     * - #catch:: Executes the given block, possibly catching a thrown object.
+     * - #fail:: Raises an exception based on the given arguments.
+     * - #raise:: Raises an exception based on the given arguments.
+     * - #throw:: Transfers control to the end of the active catch block
+     *            that is waiting for the given tag.
+     *
+     * === Strings
+     *
+     * - #format:: Returns the string resulting from applying the given format string
```

Comment:
`format` is an alias of `sprintf`.

Reply:
You updated the `sprintf` line to mention the `format` alias, so I would remove the `format` line